src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html

```diff
@@ -3,65 +3,64 @@
 </div>
 
 <div class="course-section">
-  <h2>Active Courses</h2>
   <div class="table-responsive">
     <table class="table table-striped table-bordered margin-0">
       <thead>
-        <tr class="bg-primary text-white">
-          <th>Course ID</th>
-          <th>Course Name</th>
-          <th>Creation Date</th>
-          <th>Sections</th>
-          <th>Teams</th>
-          <th>Total Students</th>
-          <th>Total Unregistered</th>
-          <th class="text-center">Action(s)</th>
-        </tr>
+      <tr class="bg-primary text-white">
```

Comment:
Do not create unnecessary indentation change here and below

Reply:
Thank you very!!! much I'll keep that in mind.